include/mbedtls/cipher_light.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * \file cipher_light.h
```

Comment:
> > In case we go for this design my question would be about the naming of this new module. This _LIGHT version is a bit different than the previous one we designed for ECP and MD. In this case this not "a part of a larger module", but it's basically a new one, so what about going for a new name (ex: cipher_dispatch or similar)?
One point to support this proposal is that even functions names are a bit different: now the prefix is mbedtls_cipher_light_ instead of mbedtls_cipher_.
> 
> Yes, absolutely, I've been thinking about it and came to the same conclusion. I'd even suggest using a name that doesn't start with cipher, in order to make it grepping for uses of Cipher easier. I had been thinking about "mini cipher", "block cipher" or "dispatch cipher"...

Thirded. I like `mbedtls_block_cipher_xxx`.